actions/lib/action.py

```diff
@@ -221,6 +221,19 @@ def do_method(self, module_path, cls, action, **kwargs):
             method_fqdn = '%s.%s.%s' % (module_path, cls, action)
             self.logger.debug('Calling method "%s" with kwargs: %s' % (method_fqdn, str(kwargs)))
 
+        if 'Filters' in kwargs.keys():
+            boto3_compliant_filters = []
+            for specified_filter in kwargs['Filters']:
+                keyval = specified_filter.split('=')
```

Comment:
Does this mean its not possible to filter on anything with a = in the value now, as it will get split? For example what happens if I can give an AWS resource a tag or value with an = in?

Reply:
Yeah you're right, I haven't thought of that. It seems that the equal sign is an allowed character in both tag names and values...
Will try to find a solution to that...

Reply:
suggestion, make `filters` an object, then you can just use the key/value pairs in that dict/object